webserver/main.py

```diff
@@ -42,25 +47,49 @@ def do_POST(self):
         print("Received input: {}".format(body))
         body_as_string = body.decode("utf-8") 
 
-        # There are always 2 oponents in fencing
-        # Each will have their point stored in a separate file
-        # When each scores, an id will be sent to the scoring server
-        # which will then increment the counter for that oponent id
-        # The dashboard will be responsible for reading the scores
-        # and declaring a winner
+        # There are always 2 oponents in fencing. Each will have their point
+        # stored in a separate file. When each scores, an id will be sent to the
+        # scoring server which will then increment the counter for that oponent
+        # id The dashboard will be responsible for reading the scores and
+        # declaring a winner.
 
         # body comes in format id=<id>
         id = body_as_string.split("=")[1]
         fencer_score_file = 'fencer#' + id + "_score.txt"
-        print("Point scored from fencer#" + id)
+        print("Touch from fencer#" + id)
+
+        global lastScoredPointTime
+        global lastScoredPointId
```

Comment:
What do these do?

Reply:
Uses the global variables instead of assuming those are local ones. The goal is to have those values persisted across API calls.
See https://www.w3schools.com/python/python_variables_global.asp